ksp_plugin_adapter/style.cs

```diff
@@ -96,8 +97,8 @@ internal static class Style {
   // It is very important to use *static* styles for these elements.  If we use
```

Comment:
Fix the comment now that the styles are not static.

Reply:
The styles are still static.